uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/federatedlogin/utils/Constants.java

```diff
@@ -0,0 +1,10 @@
+package org.cloudfoundry.identity.uaa.integration.feature.federatedlogin.utils;
+
+public class Constants {
```

Comment:
These constants are never used. Let's remove the file

Reply:
Done